microk8s-resources/default-args/certs.d/docker.io/hosts.toml

```diff
@@ -0,0 +1,4 @@
+server = "https://docker.io"
+
+[host."https://registry-1.docker.io"]
+  capabilities = ["pull", "resolve"]
```

Comment:
Please add missing new-line at the end of file

Reply:
ok